src/physicsobjects.cxx

```diff
@@ -968,6 +968,55 @@ PtCorrection_byValue(ROOT::RDF::RNode df, const std::string &corrected_pt,
         df.Define(corrected_pt, electron_pt_correction_lambda, {pt, eta});
     return df1;
 }
+/// Function to correct electron pt, based on correctionlib file
+///
+/// \param[in] df the input dataframe
+/// \param[out] corrected_pt name of the corrected tau pt to be calculated
+/// \param[in] pt name of the raw tau pt
+/// \param[in] eta name of raw tau eta
+/// \param[in] sf_barrel scale factor to be applied to electrons in the barrel
```

Comment:
thats the name of the sf right? so something like `sf_barrel_name` would maybe be better